extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -543,6 +549,11 @@ export class SchemaCompareOptionsDialog {
 				checked: this.GetSchemaCompareIncludedObjectsUtil(l),
 				label: l
 			}).component();
+
+			box.onChanged(() => {
```

Comment:
ouch we wont have check boxes any more :| changes to table pending current PR...

Reply:
rebased and moved this setting this.optionsChanged=true into onCellAction() functions